hrjobcontract/CRM/Hrjobcontract/DAO/HRJobLeave.php

```diff
@@ -197,6 +211,7 @@ static function &fieldKeys()
                 'id' => 'id',
                 'leave_type' => 'hrjobcontract_leave_leave_type',
                 'leave_amount' => 'hrjobcontract_leave_leave_amount',
+                'add_public_holidays' => 'hrjobcontract_leave_add_public_holidays',
```

Comment:
@coldrunKacper I have updated the field key in self::$_fields. Can you check if it's ok now?